apps/main-landing/src/components/products-section/components/prediction-markets-section/prediction-markets-section-actions.tsx

```diff
@@ -0,0 +1,108 @@
+import { Button } from '@idriss-xyz/ui/button';
+import { classes } from '@idriss-xyz/ui/utils';
+import { useMutation } from '@tanstack/react-query';
+import { Controller, useForm } from 'react-hook-form';
+
+type WaitlistInput = {
+  email: string;
+};
+
+export const PredictionMarketsSectionActions = () => {
+  const {
+    control,
+    handleSubmit,
+    reset,
+    formState: { errors },
+  } = useForm<WaitlistInput>({
+    defaultValues: {
+      email: '',
+    },
+  });
+
+  const mutation = useMutation({
+    mutationFn: async (data: WaitlistInput) => {
+      const response = await fetch('https://api.idriss.xyz/subscribe', {
```

Comment:
not for now but since it's a normal webapp we're not as limited as in extension, let's add TODO to use `axios` here intead of native fetch, axios has much better api's, interceptors etc... wdyt?

Reply:
good idea, I'll add TODO for now.